src/containers/JobSearch/index.tsx

```diff
@@ -67,6 +68,29 @@ const JobSearch: React.FC = () => {
     },
   ];
 
+  //Using useSpring => Animated Job number
+  const totalJob = useSpring({
+    totalJobs: jobs.allJobs?.length,
```

Comment:
Could you make `allJobs` is required? I don't think it can be nullable or undefined.